userdocs/src/usage/nodegroup-with-custom-subnet.md

```diff
@@ -0,0 +1,53 @@
+# Nodegroups with custom subnet(s)
+
+It's possible to extend an existing VPC with a new subnet and add a Nodegroup to that subnet.
+
+## Why
+
+Should the cluster run out of pre-configured IPs, it's possible to resize the existing VPC with
+a new CIDR to add a new subnet to it. To see how to do that, read this guid on AWS [Extending VPCs](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html#vpc-resize).
+
+### TL;DR
```

Comment:
You linked to the AWS guide above, do we need to have it copied it here? It doesn't really seem like it belongs to our docs. Is there a risk this will get out of date?

Reply:
There are soooo many rules to apply a new cidr block and the fact that routing is also automatically added to the main routing table. I don't know to be honest. What's the practice usually with these outside docs? I'll follow that. :)

Reply:
Create https://github.com/weaveworks/eksctl/issues/4050 to refactor the vpc document.